input/i18n/en/my-account.yaml

```diff
@@ -21,3 +21,9 @@ personalDetails:
   welcomeBack: Welcome back
   welcomeDescription: for an even better customer service please provide your customer number
   personalDetailsTitle: Your Personal Details
+shippingAddress:
+  sameBillingAddressCheckboxLabel: Use as same address for billing
+  sameDeliveryAddressCheckboxLabel: Use as same address for delivery
```

Comment:
Was this sentence in the design? As it is the address book, I would imagine a sentence like "Use as default address for billing" / "... delivery". Therefore a message key `useAsDefaultShippingAddressCheckboxLabel` / `useAsDefaultBillingAddressCheckboxLabel`